### PR TITLE
lopper: assists: xlnx_overlay_dt: Migrate the DT overlays to sugar sy…

### PR DESCRIPTION
**Pull request by NavaKishoreManne**
…ntax

Migrate the DT overlays to sugar syntax instead of hand coding overlay fragments structure.

### CODE REVIEW

File: lopper/assists/xlnx_overlay_dt.py [hunk 1]
```diff
@@ -208,13 +199,14 @@ def xlnx_generate_overlay_dt(tgt_node, sdt, options):
                 if node.name != "amba_pl":
                     if parent_tab == 1 and parent_node == node.parent:
                         plat.buf('\n')
+                        plat.buf('\nNava1')
```
Comment (kedareswararao) [inline lopper/assists/xlnx_overlay_dt.py:202]: unneeded print

File: lopper/assists/xlnx_overlay_dt.py [hunk 2]
```diff
@@ -208,13 +199,14 @@ def xlnx_generate_overlay_dt(tgt_node, sdt, options):
...
                         plat.buf('\t' * int(rt))
+                        plat.buf('\nNava2')
```
Comment (kedareswararao) [inline lopper/assists/xlnx_overlay_dt.py:204]: same here

**Comment by zeddii** [comment]
I'll wait for fixes based on the review comments before merging.

**Review by kedareswararao — APPROVED** [review]
Changes looks fine to me

**Pull request merged**